src/weapon.py

```diff
@@ -4,19 +4,55 @@
 import random
 
 class Weapon(Item):
-    def __init__(self, name, desc, value, damage):
+    def __init__(self, damage, name):
         """@ReturnType Weapon"""
-        self.___damage = damage
-        self.___name = name
+        self.__damage = damage
         """@AttributeType int"""
-        super(Weapon, self).__init__(name, desc, value)
+        super(Weapon, self).__init__(name, "A weapon to be used in combat. Equip to increase your damage.", damage)
```

Comment:
We probably don't need the Item class to have a value since we're not implementing shops or anything, so it's fine that you're passing the weapon's damage to the Item constructor as its sale value. It took me a minute to figure out what was happening here though!

Reply:
Yeah i figured value being correlated to damage amount made the most sense for now.